src/Libraries/CoreNodeModels/Enum.cs

```diff
@@ -69,11 +69,21 @@ protected AllChildrenOfType() : base(outputName)
             RegisterAllPorts();
         }
 
+        protected AllChildrenOfType(string outputName) : base(outputName)
+        {
+            RegisterAllPorts();
+        }
+
         [JsonConstructor]
         protected AllChildrenOfType(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(outputName, inPorts, outPorts)
         {
         }
 
+        [JsonConstructor]
+        protected AllChildrenOfType(string outputName, IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(outputName, inPorts, outPorts)
```

Comment:
Hi @ZiyunShang I still don't think this makes much sense because `outputName` is not a serialized property! But I don't think it hurts anything to add this.

I think your screenshot shows exactly my concern - the same node is being loaded one way, and constructed from the library another.

I think you need to change the `[JsonConstructor]` of the derived class to pass `"Class"` as the output name and call your new constructor base that you added here to get nodes to load correctly.

Reply:
@mjkkirschner Maybe I need to wait until DynamoCore Library has this change before updating.